mix.exs

```diff
@@ -3,7 +3,6 @@ defmodule Ecto.Mixfile do
 
   @version "2.0.0-dev"
   @adapters [:pg, :mysql]
```

Comment:
Depend only on `:pg` for now and update README to mention which are ready for Ecto 2.0.